tasks_examples/AlGhafa_tasks.txt

```diff
@@ -0,0 +1,11 @@
+community|Alghafa:mcq_exams_test_ar|5|1
```

Comment:
For the final version of the PR, you can remove this file :)

Reply:
Yes indeed it is just for testing purposes now